### PR TITLE
⚠️ CONFLICT! Lineage pull request for: skeleton

### PR DESCRIPTION
**Pull request by cisagovbot**
# Lineage Pull Request: CONFLICT #

[Lineage] has created this pull request to incorporate new changes found in an
upstream repository:

Upstream repository: [`https://github.com/cisagov/skeleton-docker.git`](https://github.com/cisagov/skeleton-docker.git)
Remote branch: `HEAD`

Check the changes in this pull request to ensure they won't cause issues with
your project.

The `lineage/skeleton` branch has **one or more unresolved merge conflicts**
that you must resolve before merging this pull request!

## How to resolve the conflicts ##

1. Take ownership of this pull request by removing any other assignees.

1. Clone the repository locally, and reapply the merge:

    ```console
    git clone git@github.com:cisagov/vdp-scanner-docker.git vdp-scanner-docker
    cd vdp-scanner-docker
    git remote add skeleton https://github.com/cisagov/skeleton-docker.git
    git remote set-url --push skeleton no_push
    git switch develop
    git checkout -b lineage/skeleton --track origin/develop
    git pull skeleton HEAD
    git status
    ```

1. Review the changes displayed by the `status` command.  Fix any conflicts and
   possibly incorrect auto-merges.

1. After resolving each of the conflicts, `add` your changes to the
   branch, `commit`, and `push` your changes:

    ```console
    git add Dockerfile README.md docker-compose.yml 
    git commit
    git push --force --set-upstream origin lineage/skeleton
    ```

    Note that you may _append_ to the default merge commit message
    that git creates for you, but *please do not delete the existing
    content*.  It provides useful information about the merge that is
    being performed.

1. Wait for all the automated tests to pass.

1. Check the "Everything is cool" checkbox below:

    - [X] ✌️ The conflicts in this pull request have been resolved.

1. Mark this draft pull request "Ready for review".

------------

**Note:** *You are seeing this because one of this repository's maintainers has
configured [Lineage] to open pull requests.*

For more information:

🛠 [Lineage] configurations for this project are stored in `.github/lineage.yml`

📚 [Read more about Lineage][Lineage]

[//]: # (lineage:metadata:{&quot;slayed&quot;:true})
[Lineage]: https://github.com/cisagov/action-lineage/ "Lineage GitHub Action"

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -16,42 +18,177 @@ or a given CSV in the same format with the
...
+
+The images of this container are tagged with
+[semantic versions](https://semver.org).  It is recommended that most users use
+a version tag (e.g. `:0.0.2`).
+
+| Image:tag | Description |
+|-----------|-------------|
+|`cisagov/vdp-scanner:1.2.3`| An exact release version. |
+|`cisagov/vdp-scanner:1.2`| The most recent release matching the major and minor version numbers. |
+|`cisagov/vdp-scanner:1`| The most recent release matching the major version number. |
```
Comment (jsf9k) [inline README.md]: These version numbers should be changed to agree with the current version of the Docker image.

**Review by jsf9k — CHANGES_REQUESTED** [review]
I requested one minor change.  Otherwise this looks good to me.

File: README.md (hunk 1 above)
Comment (mcdonnnj) [reply README.md]: Good thought. Fixed in 2e5b580.

**Review by jsf9k — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -16,42 +18,177 @@ or a given CSV in the same format with the
...
-```
+    ```console
+    docker-compose up --detach
+    ```
+
+## Updating your container ##
+
+### Docker Compose ###
+
+1. Pull the new image from Docker hub:
```
Comment (dav3r) [inline README.md:56]: Use preferred capitalization:
```suggestion
1. Pull the new image from Docker Hub:
```

**Review by dav3r — APPROVED** [review]
Looks great - just one tiny capitalization thang. 🧩

File: README.md (hunk 2 above)
Comment (mcdonnnj) [reply README.md:56]: That is pulled in from [cisagov/skeleton-docker](https://github.com/cisagov/skeleton-docker/blame/develop/README.md#L105) so I'd like to fix it there and let [cisagov/action-lineage](https://github.com/cisagov/action-lineage) push it down.

Comment (dav3r) [reply README.md:56]: Good call - it's going to be handled in https://github.com/cisagov/skeleton-docker/pull/65.

**Pull request merged**